pennylane/operation.py

```diff
@@ -808,7 +808,7 @@ def compute_eigvals(*params, **hyperparams):
 
         Otherwise, no particular order for the eigenvalues is guaranteed.
 
-        .. seealso:: :meth:`~.Operator.eigvals` and :func:`~.eigvals`
+        .. seealso:: :meth:`~.eigvals` and :func:`pennylane.eigvals`
```

Comment:
Can we include the namespace for both of these links, not just the last one?

Reply:
I thought it would be better to link to the `eigvals` method of the function at hand instead of the `Operator.eigvals` method. That's why I removed `Operator.`. Not sure if there is a way to make the actual class appear in the docs.

Reply:
I [did something similar](https://github.com/PennyLaneAI/pennylane/blob/f363b8cac398a9c39e36d2ec96bea00e15cb962e/pennylane/operation.py#L708) for `compute_matrix`, and I aliased the top-level function so it says `qml.` instead of `pennylane.` because that's what our users actually call. can we do the same thing here?

Reply:
also, the `~` is the thing that removes the class (or anything but the final part of the function). if you want to show the namespace/context/class, just leave out the `~`

Reply:
Oh yes, this looks better. I updated it. This is what it looks like now and it still links to the class-specific method.
![Screenshot 2023-06-22 at 5 57 02 PM](https://github.com/PennyLaneAI/pennylane/assets/42576579/a8c63fc4-2aeb-4329-948d-a18ed3a2b0aa)